src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerMessage = partnerName + (sex === 'm'
+    ? ' is my wife'
+    : ' is my husband');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {isMarried
+          ? partnerMessage
+          : 'I am not married'
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.propTypes = {
```

Comment:
add default types for component

Reply:
There is render by condition for not required props. Are default props needed? Or you mean 'types' like in TS? Maybe I don't understand.

Reply:
For some fields from the object, we can add default props, if they are not passed, then the default values will be used